README.md

```diff
@@ -1,38 +1,343 @@
+| ADMIN_LOG_CHANNEL | the channel for the  `Admin Loggers` modules to post to                                                                 |
+| JAIL_CHANNEL      | the channel you use to `jail` members                                                                                   |
+| BIRTHDAY_CHANNEL  | Used in the `birthday` command to wish members happy birthday, if this setting is not set, the command will be disabled |
+
 ## <ins>Modules</ins>
 
-There are 12 main modules in this bot, each one has its own set of permissions as defined by the`CommandSecurityModels`
-database table.
-
-| Module | Description |
-| --- | ----------- |
-| Ages | commands to get ages of accounts and servers |
-| ResourceBanner | Commands deal with banning attachments, embeds and emojis from messages |
-| Mute | Commands to mute people from servers |
-| Username | Commands to set usernames for people|
-| Memes | Commands generate memes. All commands that are marked as type 'attachment' can take both an uploaded image (has to be jpg) OR a mention |
-| Bookmarks | Commands to manage bookmarks |
-| Help | Commands to display help and info |
-| messageschedule | Commands to schedule posts to channels|
-| Miscellaneous | Miscellaneous commands |
-| closeableModules | Commands to enable or disable modules and commands |
-| Music | Commands to play music from Youtube |
-| RoleJoiner | Commands to allow users to join vanity roles|
-| Settings | Commands to change internal settings of this bot |
-
-you may use the`~help`command for more info about each command and how to call each command complete with mandatory
-arguments and examples.
+All modules in this application are defined by implementing the `ICloseableModule` interface. this applies to events
+only, not commands. Commands are defined in the own entity set contained in categories.
+
+### <ins>events</ins>
+
+#### Audit Logger
+
+The Audit logger is a non-admin styled logger designed to log quick events like when a member leaves, joins, kicked and
+banned (and by whom) and when an Auto mod event is triggered.
+
+In order for this module to work, it must be both enabled AND have the `LOG_CHANNEL` global setting populated with the
+ID of a channel
+
+This is a quick non-admin style logger that can log:
+
+- Member join
+- Member kick
+- Member leave
+- Member muted
+- Member un-muted
+- Member ban
+
+***
+
+#### Admin Loggers
+
+Admin Loggers are an embed style logger that will post realtime, detailed events to a chosen channel, these events are
+presented in a rich-embed. All triggered events contain an executor, in that will tell you who did what.
+
+for example, if a message is deleted, or a member is kicked or banned, or even if a channel was modified or created. It
+will query your servers event log and pull the person who did that action. so it will tell you who deleted the message,
```

Comment:
Unnecessary large mouth full of words, remove the last sentence and just add this:

"will query your servers event log and pull the person who did that action and send the info to the log channel" 

something like that maybe.